product_hunt_utils.py

```diff
@@ -1,84 +1,131 @@
-            'analytics_session_id.last_access': '1691052362270',
-            '_ga_WZ46833KH9': 'GS1.1.1691049905.19.1.1691052363.60.0.0',
-        }
+    '_delighted_web': '{%2271AaKmxD4TpPsjYW%22:{%22_delighted_fst%22:{%22t%22:%221689247698039%22}}}',
+    '_ga': 'GA1.1.845259885.1689247698',
+    'ajs_anonymous_id': '98a3c2ff-2922-4519-9794-e4f731553551',
+    'visitor_id': 'ddc2a4ea-9274-469e-aaba-87f9e81c7597',
+    'track_code': 'a7ccfbaf01',
+    'intercom-id-fe4ce68d4a8352909f553b276994db414d33a55c': '06cb8ae7-ecdd-49c6-84a8-d9c926edc77d',
+    'intercom-device-id-fe4ce68d4a8352909f553b276994db414d33a55c': '63d09907-e9c8-4aa3-bd10-eb96e1b656df',
+    '_hjSessionUser_3508551': 'eyJpZCI6ImVhYzIzNzQxLWQ1MWItNWE0NS1iYzc5LTcxYWExNDlhMmU5YyIsImNyZWF0ZWQiOjE2ODkyNDc2OTgyODksImV4aXN0aW5nIjp0cnVlfQ==',
+    'ajs_user_id': '5826028',
+    'g_state': '{"i_l":2,"i_p":1690215702448}',
+    'first_visit': '1691049905',
+    'first_referer': 'https://www.google.com/',
+    '_hjHasCachedUserAttributes': 'true',
+    'intercom-session-fe4ce68d4a8352909f553b276994db414d33a55c': '',
+    'analytics_session_id': '1691513461352',
+    '_hjAbsoluteSessionInProgress': '1',
+    '_hjIncludedInSessionSample_3508551': '0',
+    '_hjSession_3508551': 'eyJpZCI6ImYzYzRlZWY1LTk2ZjUtNGY0Mi1iMGI1LWRjM2M3OTNkNjE3MyIsImNyZWF0ZWQiOjE2OTE1MTM0NjY1NTUsImluU2FtcGxlIjpmYWxzZX0=',
+    'csrf_token': 'W4IQ0p2YGhOSVtgMvGBRh0hLmnqHjLR4__Zd80swYot9dbxQT5s2VPS8h0svazUhapvRws3h_6-FBqE8IVmYlw',
+    '_producthunt_session_production': 'OSf3xKPYByDwwW3YgyA0lXpmXswQe7M%2FwsyrNOLSNPgj1kgRzt2p9urAI8NjiqPKcC45uzm98eZ5K8v43wxVBsdtjMoJ8HOkh5WaAo%2Fv8z00LQ2fax0lIzyrQLVE7hbl%2BmE7MJ4z7qVq9pEFF7uS4uYTdYQD9y8Ehzvc0W%2B8big%2B%2BxNih%2FSMf%2FdtO5mvWUcNVHH9JGGeo1p0EkNorUp7ArowwwPpqjeuxqEUy1dp0jr7g1Wr8sHBZdLM55gkC1nNAmYjf4kNB6g2d0uKMZhfdordK2e2IFaUbpuZrEf%2F6BBMYiwCPz9lopb1WjjovkuqgldCYuhJhGu5O6rj%2FKtRiOFuzcqQAO6085CFgtvMFlvdxTdtj5DwDVOpJV78V7dWn0jpu%2BcdZ5FSdlEDrs0Zx716QxOaKbtNnHWr1OaNPKIp6Ft4JJ8i5rZ5UgWsStlw4nPPoTXWQNqBrbC1wCu%2BwNOyk1wlaYg8EZ%2B5wVfEg2CAmLRN2n%2FTi7qlR5gpkYwnv7igrWBtmacJY8GziQ%3D%3D--SfIMKCTzQmgIiVkM--hbRy8HbB9c8bEVneYjHwpQ%3D%3D',
+    'analytics_session_id.last_access': '1691513931850',
+    '_ga_WZ46833KH9': 'GS1.1.1691513462.24.1.1691513932.60.0.0',
+}
+
 headers = {
-            'authority': 'www.producthunt.com',
-            'accept': '*/*',
-            'accept-language': 'en-US,en;q=0.9',
-            'content-type': 'application/json',
-            'origin': 'https://www.producthunt.com',
-            'referer': 'https://www.producthunt.com/@cyriljeet',

```

Comment:
check if the request works without the "referer" header. having a specific URL in refer does feel good